src/webgpu/api/operation/storage_texture/read_only.spec.ts

```diff
@@ -580,7 +580,7 @@ g.test('basic')
       t.selectDeviceOrSkipTestCase('bgra8unorm-storage');
     }
     if (t.isCompatibility) {
-      t.skipIfTextureFormatNotUsableAsStorageTexture(t.params.format);
+      t.skipIfTextureFormatNotUsableAsStorageTexture(t.params.format, t.device);
```

Comment:
I get that the PR is incorrect but it's also not clear what problem it's trying to solve with regards to #3847. The code at line 580 should have already skipped the test if `bgra8unorm-storage` is not supported.

Reply:
IIUC the problem is the opposite: if the format is bgra8unorm, then we enable bgra8unorm-storage... but then `skipIfTextureFormatNotUsableAsStorageTexture` skips the test anyway, because the capability tables say it won't work.